### DISCUSSION TITLE
podman image trust

### DISCUSSION THREAD

**Discussion by zosocanuck** [containers/podman#23168] [General]
I'm trying to perform image signature verification with an image signed via `cosign` however the `podman image trust` CLI option doesn't seem to be available via the latest version 5.1.1 on MacOS.  Any clues?

```
podman image trust
Error: cannot use command "podman image trust" with the remote podman client
```

**Comment by Luap99** [comment]
As the error tells you this command is not supported on the remote client. Only the local linux client can use it.
You can try `podman machine ssh` on macos to enter the VM and run it "locally" there.
